internal/cache/redis/redis_cache.go

```diff
@@ -0,0 +1,64 @@
+package redis
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/go-redis/redis/v8"
+)
+
+type cache struct {
+	client *redis.Client
+}
+
+func New() *cache {
+	client := redis.NewClient(&redis.Options{
+		Addr:     config.RedisHostAndPort(),
+		Username: config.Application.Cache.Redis.User,
+		Password: config.Application.Cache.Redis.Password,
+		DB:       config.Application.Cache.Redis.DB,
+	})
+	return &cache{
+		client: client,
+	}
+}
+
+func (c *cache) FindAccountId(ctx context.Context, OrgID, AccountNumber string) (*models.Account, bool) {
+	key := OrgID + AccountNumber
+	account := models.Account{}
+
+	result := c.client.Get(ctx, key)
+	if result.Err() != nil {
+		ctxval.Logger(ctx).Warn().Err(result.Err()).Msgf("Redis returned an error for account %s", OrgID)
+		return nil, false
```

Comment:
```suggestion
		ctxval.Logger(ctx).Warn().Err(result.Err()).Msgf("Redis returned an error for account with orgID %s", OrgID)
		return nil, false
```

Reply:
I am dropping those log lines because the new version I am about to rebase to handles errors properly.